client/components/PubMenuItem/PubMenuItem.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
Split out of the `PubSelect` component currently in use on the Collection overview dashboard.